### DISCUSSION TITLE
HyperHDR to switch between two profiles.

### DISCUSSION THREAD

**Discussion by satgit62** [awawa-dev/HyperHDR#903] [General]
Hello,
I wanted to know how to switch between two color channel setting profiles on the adalight controller with HiperSerialPico.
For example, the brightness is set to 100% for the day profile and 70% for the night profiles.

**Comment by awawa-dev** [comment]
Hi
JSON API? MQTT? I think it can be automated with cron for day/night profile.

**Comment by satgit62** [comment]
Hi, that's a great idea, but under “JSON API” this is unfortunately not available for selection. Can you implement this?
![json api](https://github.com/user-attachments/assets/04ff9247-02fd-47f7-b95f-ef82220c6e4e)

**Comment by awawa-dev** [comment]
It's already implemented. You are looking into wrong method.
![obraz](https://github.com/user-attachments/assets/01360417-8efd-4e19-8ff6-90180e89b57c)

**Comment by satgit62** [comment]
Hi, Thank you
indeed, I have now looked in the right section, and will test.
